src/applications/check-in/day-of/pages/Error.jsx

```diff
@@ -3,17 +3,14 @@ import { useSelector } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 
 import { makeSelectError } from '../../selectors';
-import { useSessionStorage } from '../../hooks/useSessionStorage';
 import Wrapper from '../../components/layout/Wrapper';
 
 const Error = () => {
   const { t } = useTranslation();
-  const { getValidateAttempts } = useSessionStorage(false);
   const selectError = useMemo(makeSelectError, []);
   const { error } = useSelector(selectError);
-  const { isMaxValidateAttempts } = getValidateAttempts(window);
 
-  const validationError = isMaxValidateAttempts || error === 'max-validation';
+  const validationError = error === 'max-validation';
```

Comment:
slightly nit-picky but I find this confusing. Could we name the variable `isValidationError` to help make clear that it's a boolean, or perhaps just get rid of this line and do the conditional directly on line 29?